common/src/main/java/com/wynntils/models/gear/type/GearTier.java

```diff
@@ -13,7 +13,7 @@
 import net.minecraft.network.chat.Component;
 
 public enum GearTier {
-    NORMAL(ChatFormatting.WHITE, 0, 0.0f),
+    NORMAL(ChatFormatting.WHITE, 0, 0.0f, "common"),
```

Comment:
If Wynn is renaming the base tier to "common", maybe we should follow suite? Or are there places in our code that assume the enum name is "normal"?

Reply:
I think "common" is not used anywhere except the api, so I think it's fine to not rename this, at least until we don't see more of these renames.